src/test/resources/com/znsio/sample/e2e/features/Amazon.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Amazon Shopping Cart
+  @amazon @single-user @web
+  Scenario: user is able to add a product to shopping cart as guest user
+    Given user creates search for "Apple iPhone 13 Pro" product on Amazon
+    When user selects the first product listed in search results
```

Comment:
Given I in Amazon home page
When I search for "Iphone13" and see the details
And I add the product into the cart 
Then I should able to see the product into the cart